src/lock.rs

```diff
@@ -62,11 +62,17 @@ pub struct Lock<'a> {
     pub lock_manager: &'a LockManager,
 }
 
+#[cfg(not(feature = "tokio-comp"))]
 #[derive(Debug, Clone)]
 pub struct LockGuard<'a> {
     pub lock: Lock<'a>,
 }
 
+
+// Dropping this guard inside the context of a tokio runtime if tokio-comp is enabled 
+// will block the tokio runtime. 
+// Because of this, the guard is not compiled if tokio-comp is enabled. 
```

Comment:
```suggestion
/// Dropping this guard inside the context of a tokio runtime if `tokio-comp` is enabled 
/// will block the tokio runtime. 
///
/// Because of this, the guard is not compiled if `tokio-comp` is enabled. 
```

it seems like the comment is explaining an important behavior that could impact both users of the `LockGuard` type and developers working on the code, so lets make it a docstring